mailing_list/management/commands/make_access_levels_read_only.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+from mailgun.listserv_client import MailgunClient as ListservClient
+from mailing_list.models import MailingList
+
+
+OUTPUT_FIELDS = ['id', 'sis_course_id', 'name', 'course_code']
+
+logger = logging.getLogger(__name__)
+
+listserv_client = ListservClient()
+
+
+class Command(BaseCommand):
+    help = 'Sets access_level on all Mailgun mailing lists to read only'
+    option_list = BaseCommand.option_list + (
```

Comment:
Doesn't matter for this, but they're recommending argparse now instead of optparse.